2024-11-19 01:46:39,076 - __main__ - CRITICAL - Произошла ошибка unsupported operand type(s) for +=: 'int' and 'str'
2024-11-20 21:12:13,579 - __main__ - INFO - Инициализация
2024-11-20 21:12:14,119 - __main__ - INFO - Запуск
2024-11-20 21:12:38,810 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-20 21:12:53,989 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-20 21:14:02,020 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-20 21:17:33,861 - __main__ - INFO - Инициализация
2024-11-20 21:17:34,376 - __main__ - INFO - Запуск
2024-11-20 21:48:46,138 - __main__ - INFO - Инициализация
2024-11-20 21:48:46,808 - __main__ - INFO - Запуск
2024-11-20 21:48:55,247 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-20 21:48:56,373 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-20 21:49:17,984 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-20 21:50:24,320 - __main__ - INFO - Инициализация
2024-11-20 21:50:24,915 - __main__ - INFO - Запуск
2024-11-20 21:50:25,206 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-20 21:52:09,836 - __main__ - INFO - Инициализация
2024-11-20 21:52:10,399 - __main__ - INFO - Запуск
2024-11-20 21:52:12,617 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-20 21:52:19,740 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-20 21:59:41,719 - __main__ - INFO - Инициализация
2024-11-20 21:59:42,360 - __main__ - INFO - Запуск
2024-11-20 21:59:59,404 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-20 22:02:17,076 - __main__ - INFO - Инициализация
2024-11-20 22:02:17,629 - __main__ - INFO - Запуск
2024-11-20 22:02:21,465 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-20 22:07:03,106 - __main__ - INFO - Инициализация
2024-11-20 22:07:03,854 - __main__ - INFO - Запуск
2024-11-20 22:07:42,797 - src.services.bot.handlers.code_handler - INFO - Пользователь запросил код с почты c0ral0vii
2024-11-20 22:46:12,345 - __main__ - INFO - Инициализация
2024-11-20 22:46:12,930 - __main__ - INFO - Запуск
2024-11-20 22:47:33,786 - __main__ - INFO - Инициализация
2024-11-20 22:47:34,346 - __main__ - INFO - Запуск
2024-11-20 22:47:37,702 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-20 22:47:40,499 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @c0ral0vii
2024-11-20 22:48:03,063 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-20 22:48:22,095 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-20 22:48:23,489 - src.services.bot.handlers.admin_search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-20 22:52:36,728 - __main__ - INFO - Инициализация
2024-11-20 22:52:37,722 - __main__ - INFO - Запуск
2024-11-20 22:52:48,394 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-20 22:53:08,126 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-20 22:57:50,333 - __main__ - INFO - Инициализация
2024-11-20 22:57:50,914 - __main__ - INFO - Запуск
2024-11-20 23:00:53,675 - __main__ - INFO - Инициализация
2024-11-20 23:00:54,294 - __main__ - INFO - Запуск
2024-11-20 23:02:07,041 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-20 23:03:48,130 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-20 23:03:50,042 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @c0ral0vii
2024-11-20 23:04:03,928 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-20 23:04:40,681 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-20 23:04:42,784 - src.services.bot.handlers.admin_search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-20 23:05:54,077 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-20 23:05:56,129 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-20 23:08:53,536 - __main__ - INFO - Инициализация
2024-11-20 23:08:54,474 - __main__ - INFO - Запуск
2024-11-20 23:09:02,175 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-20 23:09:05,392 - src.services.bot.handlers.code_handler - INFO - Пользователь запросил код с почты @c0ral0vii
2024-11-20 23:09:07,315 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-20 23:37:51,426 - __main__ - INFO - Инициализация
2024-11-20 23:37:52,138 - __main__ - INFO - Запуск
2024-11-20 23:38:01,529 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-20 23:39:35,874 - __main__ - INFO - Инициализация
2024-11-20 23:39:36,440 - __main__ - INFO - Запуск
2024-11-20 23:46:11,018 - __main__ - INFO - Инициализация
2024-11-20 23:46:11,681 - __main__ - INFO - Запуск
2024-11-20 23:47:14,935 - __main__ - INFO - Инициализация
2024-11-20 23:47:15,505 - __main__ - INFO - Запуск
2024-11-20 23:49:57,242 - __main__ - INFO - Инициализация
2024-11-20 23:49:57,848 - __main__ - INFO - Запуск
2024-11-20 23:52:21,965 - __main__ - INFO - Инициализация
2024-11-20 23:52:22,659 - __main__ - INFO - Запуск
2024-11-21 00:00:38,712 - __main__ - INFO - Инициализация
2024-11-21 00:00:39,335 - __main__ - INFO - Запуск
2024-11-21 00:00:41,748 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-21 00:00:42,952 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-21 00:00:53,511 - src.services.mail.main - ERROR - de@gmail.com is not connected: b'[AUTHENTICATIONFAILED] Authentication failed.'
2024-11-21 00:00:53,513 - src.services.mail.main - INFO - Нет подключения к почтовому серверу
2024-11-21 00:01:01,198 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-21 00:01:06,632 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-21 07:55:35,355 - __main__ - INFO - Инициализация
2024-11-21 07:55:35,857 - __main__ - INFO - Запуск
2024-11-21 07:56:45,546 - __main__ - INFO - Инициализация
2024-11-21 07:56:46,216 - __main__ - INFO - Запуск
2024-11-21 07:56:51,114 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 07:57:58,591 - __main__ - INFO - Инициализация
2024-11-21 07:57:58,989 - __main__ - INFO - Запуск
2024-11-21 07:58:00,538 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 08:24:11,708 - __main__ - INFO - Инициализация
2024-11-21 08:24:12,247 - __main__ - INFO - Запуск
2024-11-21 08:25:03,525 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 08:26:03,038 - src.services.bot.handlers.admin_search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-21 08:34:34,306 - __main__ - INFO - Инициализация
2024-11-21 08:34:35,021 - __main__ - INFO - Запуск
2024-11-21 08:34:45,037 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-21 08:34:47,471 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-21 08:34:49,947 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 08:38:36,362 - __main__ - INFO - Инициализация
2024-11-21 08:38:37,039 - __main__ - INFO - Запуск
2024-11-21 08:38:42,462 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-21 08:38:46,257 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-21 08:38:47,828 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 08:43:25,837 - __main__ - INFO - Инициализация
2024-11-21 08:43:26,643 - __main__ - INFO - Запуск
2024-11-21 08:43:27,326 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 08:45:50,085 - __main__ - INFO - Инициализация
2024-11-21 08:45:50,772 - __main__ - INFO - Запуск
2024-11-21 08:45:53,569 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 08:47:09,248 - __main__ - INFO - Инициализация
2024-11-21 08:47:11,529 - __main__ - INFO - Запуск
2024-11-21 08:48:13,907 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 09:00:46,814 - __main__ - INFO - Инициализация
2024-11-21 09:00:47,983 - __main__ - INFO - Запуск
2024-11-21 09:00:50,907 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 09:14:26,292 - __main__ - INFO - Инициализация
2024-11-21 09:14:27,076 - __main__ - INFO - Запуск
2024-11-21 09:14:28,483 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 09:19:28,606 - __main__ - INFO - Инициализация
2024-11-21 09:19:29,267 - __main__ - INFO - Запуск
2024-11-21 09:19:33,756 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 09:23:17,459 - __main__ - INFO - Инициализация
2024-11-21 09:23:18,060 - __main__ - INFO - Запуск
2024-11-21 09:23:30,749 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 09:24:59,903 - __main__ - INFO - Инициализация
2024-11-21 09:25:00,596 - __main__ - INFO - Запуск
2024-11-21 09:27:16,708 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 09:30:51,501 - __main__ - INFO - Инициализация
2024-11-21 09:30:52,172 - __main__ - INFO - Запуск
2024-11-21 09:31:28,289 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 09:33:16,784 - __main__ - INFO - Инициализация
2024-11-21 09:33:17,369 - __main__ - INFO - Запуск
2024-11-21 09:33:21,799 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 09:34:49,092 - __main__ - INFO - Инициализация
2024-11-21 09:34:49,764 - __main__ - INFO - Запуск
2024-11-21 09:34:51,513 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 09:35:19,239 - src.services.bot.handlers.admin_search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-21 10:16:34,271 - __main__ - INFO - Инициализация
2024-11-21 10:16:35,014 - __main__ - INFO - Запуск
2024-11-21 10:16:39,963 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-21 10:16:40,910 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-21 10:16:43,623 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-21 10:16:45,741 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-21 10:16:58,904 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-21 10:17:10,803 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-21 10:17:18,098 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-21 10:17:21,652 - src.services.bot.handlers.admin_search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-21 10:17:25,985 - src.services.bot.handlers.admin_search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-21 10:17:59,865 - src.services.bot.handlers.admin_search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-21 10:18:00,729 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует @fddhdhf
2024-11-21 10:18:00,731 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-21 10:18:05,692 - src.services.bot.handlers.admin_search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-21 10:18:08,236 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует fddhdhf
2024-11-21 10:18:08,238 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-21 10:58:35,942 - __main__ - INFO - Инициализация
2024-11-21 11:19:16,843 - __main__ - INFO - Инициализация
2024-11-21 11:19:17,385 - __main__ - INFO - Запуск
2024-11-26 23:29:24,948 - __main__ - INFO - Инициализация
2024-11-26 23:29:25,384 - __main__ - INFO - Запуск
2024-11-26 23:38:04,299 - __main__ - INFO - Инициализация
2024-11-26 23:38:04,685 - __main__ - INFO - Запуск
2024-11-26 23:38:11,416 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-26 23:38:13,724 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @@c0ral0vii
2024-11-26 23:38:14,963 - src.services.bot.handlers.code_handler - INFO - Пользователь запросил код с почты @c0ral0vii
2024-11-26 23:38:18,741 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @@c0ral0vii
2024-11-26 23:38:19,576 - src.services.bot.handlers.code_handler - INFO - Пользователь запросил код с почты @c0ral0vii
2024-11-26 23:38:21,160 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-26 23:38:23,438 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует asd
2024-11-26 23:38:23,440 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-26 23:38:24,346 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует das
2024-11-26 23:38:24,347 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-26 23:38:27,360 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-26 23:38:30,571 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-26 23:38:44,016 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-26 23:54:17,773 - __main__ - INFO - Инициализация
2024-11-26 23:54:18,171 - __main__ - INFO - Запуск
2024-11-26 23:54:35,496 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-26 23:54:38,636 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-26 23:54:40,188 - src.services.bot.handlers.admin_export_handler - ERROR - 'User' object has no attribute 'pay_list'
Traceback (most recent call last):
  File "/app/src/services/bot/handlers/admin_export_handler.py", line 19, in export_handler
    path = await create_export_file()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/src/services/export_excel/services.py", line 8, in create_export_file
    data = await get_all_data()
           ^^^^^^^^^^^^^^^^^^^^
  File "/app/src/services/database/orm/export.py", line 28, in get_all_data
    "pay_list": user.pay_list,
                ^^^^^^^^^^^^^
AttributeError: 'User' object has no attribute 'pay_list'. Did you mean: 'pay_lists'?
2024-11-26 23:54:45,718 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-26 23:56:00,313 - src.services.bot.handlers.admin_export_handler - ERROR - 'User' object has no attribute 'pay_list'
Traceback (most recent call last):
  File "/app/src/services/bot/handlers/admin_export_handler.py", line 19, in export_handler
    path = await create_export_file()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/src/services/export_excel/services.py", line 8, in create_export_file
    data = await get_all_data()
           ^^^^^^^^^^^^^^^^^^^^
  File "/app/src/services/database/orm/export.py", line 28, in get_all_data
    "pay_list": user.pay_list,
                ^^^^^^^^^^^^^
AttributeError: 'User' object has no attribute 'pay_list'. Did you mean: 'pay_lists'?
2024-11-26 23:57:05,346 - __main__ - INFO - Инициализация
2024-11-26 23:57:05,674 - __main__ - INFO - Запуск
2024-11-26 23:57:24,959 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-27 00:02:24,944 - __main__ - INFO - Инициализация
2024-11-27 00:02:25,326 - __main__ - INFO - Запуск
2024-11-27 00:02:32,629 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 00:02:34,579 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @@c0ral0vii
2024-11-27 00:02:35,944 - src.services.bot.handlers.code_handler - INFO - Пользователь запросил код с почты @c0ral0vii
2024-11-27 00:02:37,748 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-27 00:03:09,085 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-27 00:03:10,136 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-27 00:06:59,969 - __main__ - INFO - Инициализация
2024-11-27 00:07:00,315 - __main__ - INFO - Запуск
2024-11-27 00:07:05,708 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-27 00:07:07,047 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-27 00:07:45,208 - src.services.bot.handlers.admin_search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-27 00:07:46,083 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует '123
2024-11-27 00:07:46,085 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 00:08:34,078 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 00:08:35,588 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @@c0ral0vii
2024-11-27 00:08:36,993 - src.services.bot.handlers.code_handler - INFO - Пользователь запросил код с почты @c0ral0vii
2024-11-27 13:21:15,422 - __main__ - INFO - Инициализация
2024-11-27 13:21:19,149 - __main__ - INFO - Запуск
2024-11-27 13:21:25,797 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 13:21:39,681 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @@c0ral0vii
2024-11-27 13:23:36,632 - __main__ - INFO - Инициализация
2024-11-27 13:23:37,028 - __main__ - INFO - Запуск
2024-11-27 13:23:39,953 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-27 13:23:53,199 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 13:23:54,516 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @
2024-11-27 13:31:12,883 - __main__ - INFO - Инициализация
2024-11-27 13:31:13,223 - __main__ - INFO - Запуск
2024-11-27 13:33:51,571 - __main__ - INFO - Инициализация
2024-11-27 13:33:52,195 - __main__ - INFO - Запуск
2024-11-27 13:35:34,868 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 13:35:43,879 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-27 13:35:51,456 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 13:35:53,427 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-27 13:35:54,488 - src.services.bot.handlers.admin_export_handler - ERROR - (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedTableError'>: relation "users" does not exist
[SQL: SELECT users.id, users.user_id, users.username, users.user_number, users.telegram_user, users.need_pay_date, users.pay_date, users.activated_date, users.stay_on_pause, users.serial_number, users.email, users.password, users.pay_lists, users.created, users.updated 
FROM users]
(Background on this error at: https://sqlalche.me/e/20/f405)
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/dialects/postgresql/asyncpg.py", line 510, in _prepare_and_execute
    prepared_stmt, attributes = await adapt_connection._prepare(
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/dialects/postgresql/asyncpg.py", line 756, in _prepare
    prepared_stmt = await self._connection.prepare(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/asyncpg/connection.py", line 635, in prepare
    return await self._prepare(
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/asyncpg/connection.py", line 653, in _prepare
    stmt = await self._get_statement(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/asyncpg/connection.py", line 432, in _get_statement
    statement = await self._protocol.prepare(
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "asyncpg/protocol/protocol.pyx", line 165, in prepare
asyncpg.exceptions.UndefinedTableError: relation "users" does not exist

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 1967, in _exec_single_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/default.py", line 941, in do_execute
    cursor.execute(statement, parameters)
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/dialects/postgresql/asyncpg.py", line 568, in execute
    self._adapt_connection.await_(
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 132, in await_only
    return current.parent.switch(awaitable)  # type: ignore[no-any-return,attr-defined] # noqa: E501
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 196, in greenlet_spawn
    value = await result
            ^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/dialects/postgresql/asyncpg.py", line 546, in _prepare_and_execute
    self._handle_exception(error)
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/dialects/postgresql/asyncpg.py", line 497, in _handle_exception
    self._adapt_connection._handle_exception(error)
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/dialects/postgresql/asyncpg.py", line 780, in _handle_exception
    raise translated_error from error
sqlalchemy.dialects.postgresql.asyncpg.AsyncAdapt_asyncpg_dbapi.ProgrammingError: <class 'asyncpg.exceptions.UndefinedTableError'>: relation "users" does not exist

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/app/src/services/bot/handlers/admin_export_handler.py", line 19, in export_handler
    path = await create_export_file()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/src/services/export_excel/services.py", line 8, in create_export_file
    data = await get_all_data()
           ^^^^^^^^^^^^^^^^^^^^
  File "/app/src/services/database/orm/export.py", line 11, in get_all_data
    result = await session.execute(stmt)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/ext/asyncio/session.py", line 461, in execute
    result = await greenlet_spawn(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 201, in greenlet_spawn
    result = context.throw(*sys.exc_info())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 2362, in execute
    return self._execute_internal(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/session.py", line 2247, in _execute_internal
    result: Result[Any] = compile_state_cls.orm_execute_statement(
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/orm/context.py", line 305, in orm_execute_statement
    result = conn.execute(
             ^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 1418, in execute
    return meth(
           ^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/sql/elements.py", line 515, in _execute_on_connection
    return connection._execute_clauseelement(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 1640, in _execute_clauseelement
    ret = self._execute_context(
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 1846, in _execute_context
    return self._exec_single_context(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 1986, in _exec_single_context
    self._handle_dbapi_exception(
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 2355, in _handle_dbapi_exception
    raise sqlalchemy_exception.with_traceback(exc_info[2]) from e
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/base.py", line 1967, in _exec_single_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/engine/default.py", line 941, in do_execute
    cursor.execute(statement, parameters)
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/dialects/postgresql/asyncpg.py", line 568, in execute
    self._adapt_connection.await_(
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 132, in await_only
    return current.parent.switch(awaitable)  # type: ignore[no-any-return,attr-defined] # noqa: E501
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 196, in greenlet_spawn
    value = await result
            ^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/dialects/postgresql/asyncpg.py", line 546, in _prepare_and_execute
    self._handle_exception(error)
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/dialects/postgresql/asyncpg.py", line 497, in _handle_exception
    self._adapt_connection._handle_exception(error)
  File "/usr/local/lib/python3.12/site-packages/sqlalchemy/dialects/postgresql/asyncpg.py", line 780, in _handle_exception
    raise translated_error from error
sqlalchemy.exc.ProgrammingError: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedTableError'>: relation "users" does not exist
[SQL: SELECT users.id, users.user_id, users.username, users.user_number, users.telegram_user, users.need_pay_date, users.pay_date, users.activated_date, users.stay_on_pause, users.serial_number, users.email, users.password, users.pay_lists, users.created, users.updated 
FROM users]
(Background on this error at: https://sqlalche.me/e/20/f405)
2024-11-27 13:36:39,561 - __main__ - INFO - Инициализация
2024-11-27 13:36:39,954 - __main__ - INFO - Запуск
2024-11-27 13:39:19,378 - __main__ - INFO - Инициализация
2024-11-27 13:39:19,897 - __main__ - INFO - Запуск
2024-11-27 13:39:25,547 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 13:39:27,418 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @
2024-11-27 13:39:51,118 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-27 13:39:54,512 - src.services.bot.handlers.admin_import_handler - INFO - Импорт @c0ral0vii
2024-11-27 13:40:54,126 - src.services.bot.handlers.admin_search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-27 13:40:55,522 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует @@c0ral0vii
2024-11-27 13:40:55,523 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:40:57,670 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует @c0ral0vii
2024-11-27 13:40:57,671 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:41:00,808 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует c0ral0vii
2024-11-27 13:41:00,809 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:43:30,345 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует tankist613
2024-11-27 13:43:30,347 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:43:33,955 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует @tankist613
2024-11-27 13:43:33,956 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:43:41,244 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует M98_FaZa
2024-11-27 13:43:41,245 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:43:45,586 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует @M98_FaZa
2024-11-27 13:43:45,587 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:43:47,425 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует @@M98_FaZa
2024-11-27 13:43:47,427 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:47:47,116 - __main__ - INFO - Инициализация
2024-11-27 13:47:47,598 - __main__ - INFO - Запуск
2024-11-27 13:47:49,206 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 13:47:51,282 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @@c0ral0vii
2024-11-27 13:48:05,889 - src.services.bot.handlers.admin_handler - INFO - Пользователь вошёл в админ панель - @c0ral0vii, ID: 944360812
2024-11-27 13:48:58,210 - src.services.bot.handlers.admin_search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-27 13:51:24,978 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 13:51:27,060 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-27 13:51:29,532 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует c0ral0vii
2024-11-27 13:51:29,533 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:51:31,702 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует @c0ral0vii
2024-11-27 13:51:31,703 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:51:50,964 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует Найти в базе
2024-11-27 13:51:50,966 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:51:51,930 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует 7 904 902 4696
2024-11-27 13:51:51,931 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:52:04,830 - src.services.database.orm.users - CRITICAL - Не удаётся получить пользователя или такого серийного номера не существует Получить код с почты
2024-11-27 13:52:04,832 - src.services.bot.handlers.admin_search_handler - CRITICAL - ID-944360812, @c0ral0vii---Not found, Вы не правильно ввели номер или его не существует
2024-11-27 13:52:27,179 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 13:52:28,512 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @@c0ral0vii
2024-11-27 13:54:54,285 - __main__ - INFO - Инициализация
2024-11-27 13:54:54,707 - __main__ - INFO - Запуск
2024-11-27 13:55:10,897 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 13:55:13,263 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @c0ral0vii
2024-11-27 13:56:36,534 - __main__ - INFO - Инициализация
2024-11-27 13:56:36,807 - __main__ - INFO - Запуск
2024-11-27 13:56:41,519 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 13:56:43,459 - src.services.database.orm.users - CRITICAL - Не удалось получить пользователя @c0ral0vii
2024-11-27 13:59:24,045 - __main__ - INFO - Инициализация
2024-11-27 13:59:24,523 - __main__ - INFO - Запуск
2024-11-27 13:59:41,622 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 13:59:44,207 - src.services.mail.main - INFO - Сообщения отсутствуют
2024-11-27 14:03:35,354 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 14:03:47,944 - src.services.mail.main - INFO - Сообщения отсутствуют
2024-11-27 14:04:54,413 - __main__ - INFO - Инициализация
2024-11-27 14:04:58,601 - __main__ - INFO - Запуск
2024-11-27 14:05:15,091 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 14:05:29,319 - src.services.bot.handlers.code_handler - INFO - Пользователь запросил код с почты c0ral0vii
2024-11-27 23:38:56,639 - __main__ - INFO - Инициализация
2024-11-27 23:38:57,063 - __main__ - INFO - Запуск
2024-11-27 23:39:29,964 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 23:39:31,297 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-27 23:41:01,903 - __main__ - INFO - Инициализация
2024-11-27 23:41:02,263 - __main__ - INFO - Запуск
2024-11-27 23:41:04,125 - src.services.bot.handlers.start_handler - INFO - Пользователь @c0ral0vii, ID 944360812 - /start
2024-11-27 23:41:05,423 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-27 23:41:15,210 - src.services.bot.handlers.code_handler - INFO - Запрошен серийный номер -- KIT400594583
2024-11-27 23:41:21,834 - src.services.bot.handlers.search_handler - INFO - Начал поиск - @c0ral0vii, ID: 944360812
2024-11-27 23:41:26,554 - src.services.bot.handlers.code_handler - INFO - Запрошен серийный номер -- 123
